Documentation/ApiOverview/GlobalValues/Typo3ConfVars/FE.rst

```diff
@@ -767,15 +767,21 @@ $GLOBALS['TYPO3_CONF_VARS']['FE']['contentRenderingTemplates']
 $GLOBALS['TYPO3_CONF_VARS']['FE']['ContentObjects']
 ===================================================
 
-.. confval:: ContentObjects
+.. versionchanged:: 12.0
 
-   :Path: $GLOBALS['TYPO3_CONF_VARS']['FE']
-   :type: array
-   :Default: []
+   The global variable `$GLOBALS['TYPO3_CONF_VARS']['FE']['ContentObjects']` is
+   not interpreted anymore.
```

Comment:
```suggestion
   not used anymore.
```

Reply:
you can still use it if you want to make your extension compatible with both v11 and 12. But version 12 will not interpret it. Therefore I think my sentence is more specific

Reply:
If that was the idea, then we should explicitly mention this.

And my suggestion was only wording, it didn't have a different meaning. ;-)

Reply:
@mbrodala  Ok, I reformulated the sentence once more, better like that?

Reply:
@mbrodala do my changes resolve this comment?